packages/trader/src/Modules/CFD/Containers/cfd-password-manager-modal.jsx

```diff
@@ -28,7 +28,7 @@ import {
 } from '@deriv/shared';
 import { connect } from 'Stores/connect';
 import CFDStore from 'Stores/Modules/CFD/cfd-store';
-import TradingPasswordManager from './trading-password-manager.jsx';
+import TradingPasswordManager from './trading-password-manager';
```

Comment:
ok, in this PR I see that the extension is removed because we switch from `jsx` to `tsx`. So I wonder if we can keep those extensions when connected to local files? (adding extensions is more Web consensus, and node-js introduced no extensions way of importing modules)

Reply:
All of the extensions will be removed at the end of the refactoring process. I am removing the extension just for the components that are converted to ts. I can differentiate the components that are converted in the imports and the ones that are not but at the end all of them will have their extensions removed.